src/figuresClasses.ts

```diff
@@ -1,11 +1,81 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
 
-export class Triangle implements Figure {}
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const maxSide: number = Math.max(this.a, this.b, this.c);
+
+    if (
+      this.a <= 0 ||
+      this.b <= 0 ||
+      this.c <= 0 ||
+      maxSide >= this.a + this.b + this.c - maxSide
+    ) {
+      throw new Error(
+        'Triangle sides should be positive numbers ' +
+          'and satisfy the triangle inequality',
+      );
```

Comment:
[CODE STYLE] - When throwing an error, it's good practice to provide a detailed description. Here the error message is detailed enough, but make sure to keep this in mind for consistency across all error messages in the code.